test/sql.test.js

```diff
@@ -281,7 +281,7 @@ describe('sql connector', function() {
   });
 
   it('builds UPDATE', function() {
-    var sql = connector.buildUpdate('customer', {name: 'John'}, {vip: false});
+    var sql = connector._buildUpdate('customer', {name: 'John'}, {vip: false});
```

Comment:
Hi @raymondfeng,

Now I have added `buildUpdate` as an alias of `_buildUpdate`; but I'm not too sure how to have a deprecation notice; do you have any idea?

Thanks a lot Raymond :-)